language_detector/main.py

```diff
@@ -2,10 +2,31 @@
 
 """This is the entry point of the program."""
 
-from languages import LANGUAGES
+from .languages import LANGUAGES
 
 
 def detect_language(text, languages=LANGUAGES):
     """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    
+    ECounter = 0
```

Comment:
Our group had a similar approach of hard-coding the english, spanish, and german counters. However, Santiago noted that our code was not dynamic enough. If there are more languages added to the LANGUAGES list, then our program would not work. Try using a For Loop to iterate through the languages, and that should help condense your code even more. Otherwise, Great job!